module/VuFind/tests/fixtures/marc/multibyte_indicators.mrc

```diff
@@ -0,0 +1 @@
+02056cam a22003974i 450000100160000000500170001600800410003301000170007402000300009102000270012102000260014804000240017404100080019805000250020608200170023108400140024808400140026209000260027610000420030224500990034425000200044326400660046330000420052933600280057133700490059933800250064850400730067352005570074665000320130365000220133565000130135765000140137077601660138485200610155095200470161199480305390624620220905075611.0180406s2019    xxua     b    001 0 eng c  a  2018015935  a9781433829734 (softcover)  a1433829738 (softcover)  z9781433829789 (ebook)  aDLCbfinerdadFI-R  aeng  aPE1408b.S48787 2019  a808/.042223 �a38.32ykl  a16.72ykl  a(Alma)9948030539062461 aSilvia, Paul J.,d1976-ekirjoittaja.1 aHow to write a lot :ba practical guide to productive academic writing /cPaul J. Silvia, PhD.  aSecond edition. 1aWashington, DC :bAmerican Psychological Association,c[2019]  axiii, 145 sivua :bkuvitettu ;c21 cm  atekstibtxt2rdacontent  akäytettävissä ilman laitettabn2rdamedia  anidebnc2rdacarrier  aSisältää bibliografisia viitteitä (sivut 135-138) ja hakemiston.  a"All academics need to write, and many struggle to finish their dissertations, articles, books, or grant proposals. Writing is hard work and can be difficult to wedge into a frenetic academic schedule. This revised and updated edition of Paul Silvia's popular guide provides practical, lighthearted advice to help academics overcome common barriers and become productive writers. Silvia's expert tips have been updated to apply to a wide variety of disciplines, and this edition has a new chapter devoted to grant and fellowship writing"--cJulkaisija.  aEnglish languagexRhetoric.  aAcademic writing.12aWriting.22aResearch. 8iVerkkoaineisto:aSilvia, Paul  J., 1976- author.tHow to write a lotbSecond edition.dWashington, DC : American Psychological Association, [2019]z97814338297898 bminervacKTAiLaihTutkimuskirjallisuus82265015310006246  bminervacKTAiLaioTutkimuskirjallisuus SIL
```

Comment:
You have added a fixture but not a test; is something missing?

Reply:
Oops. Now added. I also realized that encoding wasn't working properly either, so that's now fixed as well.